packages/big-design/src/components/Pagination/Pagination.tsx

```diff
@@ -18,6 +18,9 @@ export interface PaginationProps extends MarginProps {
   totalItems: number;
   onPageChange(page: number): void;
   onItemsPerPageChange(range: number): void;
+  previousPageLabel?: string;
+  nextPageLabel?: string;
+  getRangeLabel?(start: number, end: number, totalItems: number): string;
```

Comment:
Overall I think this is great! I'd love to sort of condense the names just a bit and add one more if that cool? Generally, I'd like this to align with Atlaskit: https://atlassian.design/components/pagination/code
```suggestion
  label?: string;
  previousLabel?: string;
  nextLabel?: string;
  getRangeLabel?(start: number, end: number, totalItems: number): string;
```

The new addition `label` would be for `aria-label="pagination"`.